site/docs/v1/tech/apis/applications.adoc

```diff
@@ -65,6 +66,72 @@ include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_application-response-body.adoc[]
 
+== Create a New Application From an Existing Application
+
+This API is used to create an Application from an existing Application.
+
+Specifying an Id on the URI will instruct FusionAuth to use that Id when creating the Application. Otherwise, FusionAuth will generate an Id for the Application.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create an Application with a generated Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/application#
+--
+
+include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-create-operation.adoc[]
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create an Application with the given Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/application/`\{applicationId\}`#
+--
+
+==== Request Parameters
+
+[.api]
+[field]#applicationId# [type]#[UUID]# [optional]#Optional# [default]#defaults to a generated UUID#::
+The Id to use for the new Application. If not specified a secure random UUID will be generated.
+
+include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-create-operation.adoc[]
+
+==== Request Body
+
+[.api]
+[field]#application.name# [type]#[String]# [required]#Required#::
+The name of the Application.
+
+[field]#sourceApplicationId# [type]#[UUID]# [required]#Required# [since]#Available since 1.43.0#::
+The Id of an existing Application from which a copy will be made.
++
+A unique [field]#application.name# is required.
++
+The `oauthConfiguration.clientSecret` and each role Id will be cleared so new values can be generated. All other values will be copied from the source Application.
+
+.Example request JSON
```

Comment:
Move to a file like the other request examples, then you'll get formatting for free as well.

Reply:
Ok refactored all "request bodies" to their own includes file and added includes for the JSON files as well.